lib/vdsm/storage/exception.py

```diff
@@ -1676,6 +1676,15 @@ def __init__(self, pvs, vgname):
 # NOTE: code is 621 used by LVMCommandError above.
 
 
+class UnexpectedLvmConfigOutput(StorageException):
+    code = 622
+    msg = "Unexpected LVM config output. Only single config value can be " \
+          "queried."
```

Comment:
Looking at the code and the places you raise this error, this message:

    Only single config value can be queried

Does not look relevant. Maybe this is a leftover from previous version?

Also, this is raised from lvmconf, running always as root from vdsm-tool. Do we really need a public error for this or we can use a module error, used only for
logging error in vdsm-tool?

Reply:
Sure, it can be module exception. Done.